### PR TITLE
Added hero attribute points

### PR DESCRIPTION
**Pull request by ijnji**

### CODE REVIEW

File: static/roles.js [hunk 1]
```diff
@@ -43,7 +43,8 @@ const Roles =
         if (!this.heroes || !this.selfHeroes.length) return [];
         const roles = new Map();
         for (const sh of this.selfHeroes) {
-          for (const role of this.heroes[sh.hero_id].roles) {
```
Comment (cjiang) [inline static/roles.js:46]: This no longer shows str/agi/int, ranged/melee on the UI.

**Review by cjiang — CHANGES_REQUESTED** [review]

File: static/roles.js (hunk 1 above)
Comment (ijnji) [reply static/roles.js:46]: True. That was intentional.  I feel like those aren't roles.  The data is still there for a separate graph.

Comment (ijnji) [reply static/roles.js:46]: Added back with following commit.

**Review by cjiang — APPROVED** [review]

**Comment by cjiang** [comment]
Thanks!

**Pull request merged**